src/features/chat/components/ChatTopBar.tsx

```diff
@@ -0,0 +1,51 @@
+import { UserProfileWidget } from "shared/components";
+import useTheme from "features/theme/useTheme";
+import { Box, IconButton } from "@mui/material";
+import { Phone, Videocam, Info } from "@mui/icons-material";
+
+type ChatTopBarProps = {
+  profilePicture?: string;
+  firstName: string;
+  lastName: string;
+};
+
+function ChatTopBar({ profilePicture, firstName, lastName }: ChatTopBarProps) {
+  const {
+    themeColor: { backgroundColor },
+  } = useTheme();
+
+  return (
+    <Box
+      component="div"
+      sx={{
+        display: "flex",
+        flexDirection: "row",
```

Comment:
Nit: flexDirection is row by default for display flex, it's not necessary to specify it